arch/risc-v/src/mpfs/Kconfig

```diff
@@ -137,6 +137,14 @@ config MPFS_EMMCSD_MUX_GPIO
 	---help---
 		External mux GPIO between e.MMC and SD-card
 
+config MPFS_ROMFS_MOUNT
+	bool "Mount the ROMFS file system"
+		depends on FS_ROMFS
```

Comment:
Please fix double tab on this depends line

Reply:
Done